openfisca_italy/variables/statistiche.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+
+# This file defines the variables of our legislation.
+# A variable is property of a person, or an entity (e.g. a household).
+# See http://openfisca.org/doc/variables.html
+
+# Import from openfisca-core the common python objects used to code the legislation in OpenFisca
+from openfisca_core.model_api import *
+# Import the entities specifically defined for this tax and benefit system
+from openfisca_italy.entita import *
+
+# COMMENT THIS SECTION BECAUSE WHEN TRY TO RUN TEST RETURN: 
+# AssertionError: Unexpected attributes in definition of variable "total_benefits": 'total_benefits'
+# The error returns when try to run this command: openfisca-run-test openfisca_italy\tests\eta.yaml
+
+
+'''
+class total_benefits(Variable):
+    value_type = float
+    entity = Household
+    definition_period = MONTH
+    label = "Somma dei benefici percepiti da una famiglia"
+    reference = "https://stats.gov.example/benefits"
+    def total_benefits(household, period, parameters):
```

Comment:
This doesn't work because a `formula` but be named `formula`, or `formula_2016` if it only starts from 2016.
Otherwise OpenFisca doesn't know what this function is and just raises an error.